.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,45 @@
+name: Lint and Test
+
+on: pull_request
+
+jobs:
+
+  format:
```

Comment:
Since we're using pre-commit, these can all be combined into a single job! It just needs to `pip install pre-commit` and then run `pre-commit run --all-files`

Reply:
Right on! I'll make that change.